store/src/main/java/com/zextras/mailbox/servlet/MetricsServletModule.java

```diff
@@ -0,0 +1,15 @@
+package com.zextras.mailbox.servlet;
+
+import com.google.inject.servlet.ServletModule;
+import com.zextras.mailbox.metric.SoapCollector;
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.exporter.MetricsServlet;
+
+public class MetricsServletModule extends ServletModule {
+
+  @Override
+  protected void configureServlets() {
+    CollectorRegistry.defaultRegistry.register(new SoapCollector());
+    serve("/metrics").with(new MetricsServlet());
+  }
+}
```

Comment:
Btw I have found a more idiomatic way to define this, using injection
```suggestion
  @Override
  protected void configureServlets() {
    serve("/metrics").with(MetricsServlet.class);
  }

  @Provides
  @Singleton
  public SoapCollector provideSoapCollector() {
    final SoapCollector soapCollector = new SoapCollector();
    return soapCollector;
  }

  @Provides
  @Singleton
  public CollectorRegistry provideCollector() {
    return CollectorRegistry.defaultRegistry;
  }

  @Provides
  @Singleton
  public MetricsServlet provideMetricsServlet(CollectorRegistry collectorRegistry, SoapCollector soapCollector) {
  collectorRegistry.register(soapCollector);
    return new MetricsServlet(collectorRegistry);
  }
```
If nobody has nothing against current changes I will consider adding it in the next iteration :sunglasses: